src/sentry/static/sentry/app/icons/icon-trash.svg

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
i'm not actually using this, but I figure we'll need a trash icon one of these days.

Reply:
🗑 cool, doesn't hurt